cmd/falco/snippet.go

```diff
@@ -32,6 +32,14 @@ acl {{ .Name }} {
 }
 `
 
+var backendTemplate = `
+backend F_{{ .Name }} {}
+`
+
+var directorTemplate = `
+director {{ .Name }} {{ .Type | printtype }} {}
```

Comment:
falco lints required director properties according to its type (random, fallback, etc..) then name only director may raise a lint error (required properties are declared [here](https://github.com/ysugimoto/falco/blob/main/linter/helper.go#L52)) and fastly docs is [here](https://developer.fastly.com/reference/vcl/declarations/director/).

So I suggest that you can add additional properties to pass the lint.
I assumed from [your implementation](https://github.com/ysugimoto/falco/pull/95/files#diff-a254485a9e67552a5ae5b639419c368b8bbd1d2a03aa36779fb8a2fbdab5c7c9R38-R42) that the shield director type is one of `random`, `client`, or `hash` so the `weight` required property should be declared and some example value like `1` is enough.

Reply:
Ah sorry I misunderstood, the `.weight` property is required inside the backend definition.
However, falco lints director has at least one backend definition so the director definition should be:

```go
var directorTemplate = `
director {{ .Name }} {{ .Type | printtype }} {
  { .backend = F_{{ .Name }}; .weight = 1; }
}
`
```

Then linter will be passed.